llvm/include/llvm/Analysis/TargetTransformInfo.h

```diff
@@ -1907,6 +1907,10 @@ class TargetTransformInfo {
   /// pad to. Default is no padding.
   unsigned getNumBytesToPadGlobalArray(unsigned Size, Type *ArrayType) const;
 
+  /// \return Returns true if vectorizing 4 x i8s into an i32 is possible.
+  /// Currently only used by the SLP vectorizer.
+  bool canVectorizei8s() const;
```

Comment:
IMHO this is not really a good function names, especially the `i8s` part.

Reply:
It's been removed.